code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -262,20 +262,61 @@
 	description = "Coffee is a brewed drink prepared from roasted seeds, commonly called coffee beans, of the coffee plant."
 	color = "#482000" // rgb: 72, 32, 0
 	nutriment_factor = 0
-	overdose_threshold = 80
+	addiction_threshold = 30
+	overdose_threshold = 61
 	taste_description = "bitterness"
 	glass_icon_state = "glass_brown"
 	glass_name = "glass of coffee"
 	glass_desc = "Don't drop it, or you'll send scalding liquid and glass shards everywhere."
 
 /datum/reagent/consumable/coffee/overdose_process(mob/living/M)
-	M.Jitter(5)
+	var/datum/disease/D = new /datum/disease/heart_failure
```

Comment:
```suggestion
	var/datum/disease/heart_failure/D = /datum/disease/heart_failure
```
Don't waste effort creating a new heart failure disease everytime if the check below won't pass (typepaths work just fine for it).

Reply:
Use a check for current cycle if you want to run a limited bit of code, or use overdose_start() instead

Reply:
overdose_start() is only called one time, you can cheese it with just a little bit of corazone afterward.

Reply:
You want it to constantly apply the status? What if someone treats you with cora and assumes you're fine, or assumes it's bugged?

Reply:
Constantly applying while overdosing seems like a good idea, as the heart will be in constant overdrive. Will apply the above changes and see how it works out though!